torch/_dynamo/variables/torch.py

```diff
@@ -381,6 +381,25 @@ def get_state_from_generator():
                 **options,
             )
         else:
+            any_symints_or_symfloats = any(
+                [isinstance(x, DynamicShapeVariable) for x in args]
+            )
+            all_ints_or_floats = all(
+                [
+                    isinstance(
+                        x, (variables.ConstantVariable, variables.DynamicShapeVariable)
+                    )
+                    for x in args
+                ]
+            )
+            if any_symints_or_symfloats and all_ints_or_floats:
```

Comment:
so this basically checks for a case where there are no tensor inputs?

also `all_ints_or_floats` technically means `all_ints_or_floats_or_symints_or_symfloats` right?  why is the `any` variable even necessary?

wait, i almost missed this- is there a diff between DynamicShapeVariable and variables.DynamicShapeVariable?

Reply:
So this was wrong - I just updated this with a way more conservative version. But the idea behind this change was:

(1) Now that we're allowing graphs with input mutations into aot autograd, more graphs are going through our stack and (potentially) exposing more bugs

(2) One bug that showed up was from this code in `hf_Longformer`:

```
              hidden_states = hidden_states.view(
                  hidden_states.size(0),
                  torch.div(hidden_states.size(1), (window_overlap * 2), rounding_mode="trunc"),
                  window_overlap * 2,
                  hidden_states.size(2),
              )
```

Where we're calling torch.div(symint, int). `torch.div()` always creates a tensor, and we're passing that tensor as a size argument into a view call, which immediately calls `.item()` on the created tensor.

That breaks today, because fake tensor doesn't support const-propagating symints that are stored as tensor data. So I added this graph break as a dumb workaround for now.